### PR TITLE
Add public bookcase icon

### PR DESCRIPTION
**Pull request by dvdmrtnz**
* Add amenity=public_bookcase icon. Closes https://github.com/organicmaps/organicmaps/issues/4374

| Day | Night |
|-|-|
|  |  |

* Also modified library and bookshop icon for consistency

### CODE REVIEW

**Comment by vng** [comment]
LGTM. To discuss - is the book's icon recognizable?

**Comment by Jean-BaptisteC** [comment]
Maybe we can reuse bookstore icon with hat?

File: data/styles/clear/include/Icons.mapcss [hunk 1]
```diff
@@ -1729,6 +1729,10 @@ area|z18-[amenity=bbq]
 node|z16-[man_made=survey_point]
 {icon-image: survey_point-m.svg;font-size: 11;}
 
+node|z19-[amenity=public_bookcase],
+area|z19-[amenity=public_bookcase]
+{icon-image: bookcase-m.svg;font-size: 11;}
```
Comment (pastk) [inline data/styles/clear/include/Icons.mapcss:1734]: let's add it to z18?

**Comment by pastk** [comment]
> LGTM. To discuss - is the book's icon recognizable?

I couldn't immediately tell its a book, to me it looks like a kiosk or a shelter maybe :)

**Comment by RedAuburn** [comment]
Here's a rough mock-up of it with the old book style:
bitmap

**Comment by RedAuburn** [comment]
here's a rough mock-up using the old book style:
bitmap

i think the new filled icon might be a bit too hard to distinguish, it looks like it could be saloon doors etc.

whatever is chosen, it's a good idea to unify the style in the library icon 👍

**Comment by dvdmrtnz** [comment]
Changed to current bookshop icon. Seems to be more recognizable.

IMG_2710
IMG_2708

File: data/styles/clear/include/Icons.mapcss (hunk 1 above)
Comment (dvdmrtnz) [reply data/styles/clear/include/Icons.mapcss:1736]: Moved up to z18-

**Review by RedAuburn — APPROVED** [review]
LGTM! 
Nice clean SVG files :)

**Review by pastk — APPROVED** [review]

**Comment by biodranik** [comment]
It's acceptable IMO. Also, maybe try something like this? 
image

**Comment by dvdmrtnz** [comment]
> It's acceptable IMO. Also, maybe try something like this? image

Okay, I gave it a go and designed a similar bookshelf icon. Let me know what you think.

**Comment by RedAuburn** [comment]
This type of books looks good 👍 could you also add the roof from the previous versions?

**Comment by Misalf-git** [comment]
Hmm, I'd prefer the opened book, similar to the bookstore.

**Comment by dvdmrtnz** [comment]
> This type of books looks good 👍 could you also add the roof from the previous versions?

With roof

**Comment by dvdmrtnz** [comment]
> > This type of books looks good 👍 could you also add the roof from the previous versions?
> 
> With roof

IMG_2731

**Comment by dvdmrtnz** [comment]
So which one is your favorite, guys?

A. 
B. 
C.

**Comment by pastk** [comment]
I like A the most, its the most recognizable IMO!
For me B looks a bit like bowling pins :)

**Comment by biodranik** [comment]
Agree. For the ideal comparison, they all should be nearby with other icons on a map screenshot )

**Comment by dvdmrtnz** [comment]
> Agree. For the ideal comparison, they all should be nearby with other icons on a map screenshot )

There are screenshots of all of them in previous comments in this PR

**Comment by vng** [comment]
I like A variant. Clean and matches current "books" categories.

**Pull request merged**